src/main/java/org/opentripplanner/gtfs/mapping/RouteMapper.java

```diff
@@ -35,10 +50,21 @@ Route map(org.onebusaway.gtfs.model.Route orginal) {
 
   private Route doMap(org.onebusaway.gtfs.model.Route rhs) {
     var lhs = Route.of(AgencyAndIdMapper.mapAgencyAndId(rhs.getId()));
-
+    I18NString longName = null;
+    if (rhs.getLongName() != null) {
+      longName =
+        this.translationHelper != null
```

Comment:
In what case is `translationHelper` null?

Reply:
Actually `translationHelper`  is null only in Unit tests, there is a constructor where it is not initialized, and currently is only used in test classes. I think it would be good idea to refactor those test classes to have translationHelpers as well. Then this check is not needed anymore.

Reply:
Yes, that would be good. It would make reading this code easier.

Reply:
Done.